rust/xaynet-core/src/mask/object/serialization.rs

```diff
@@ -249,6 +256,38 @@ impl FromBytes for MaskMany {
 
         Ok(MaskMany { data, config })
     }
+
+    fn from_byte_stream<I: Iterator<Item = u8> + ExactSizeIterator>(
+        iter: &mut I,
+    ) -> Result<Self, DecodeError> {
+        let config = MaskConfig::from_byte_stream(iter)?;
+        if iter.len() < 4 {
+            return Err(anyhow!("byte stream exhausted"));
+        }
+        let numbers = u32::from_byte_stream(iter)
+            .context("failed to parse the number of item in mask object")?;
```

Comment:
In the code we use `numbers` because it's short, but it really refers to the number of weights in the mask. For errors, I think it's a little clearer if we're more explicit.

Reply:
ah true. then I suggest
```suggestion
            .context("failed to parse the number of items in mask object")?;
```

Reply:
oops. Yes thank you